tests/common.py

```diff
@@ -591,6 +592,12 @@ def load_json_object_fixture(
     return json_loads_object(load_fixture(filename, integration))
 
 
+def load_ssdp_fixture(filename: str, integration: str | None = None) -> SsdpServiceInfo:
+    """Load a SsdpServiceInfo object from a fixture."""
+    kwargs = load_json_object_fixture(filename, integration)
+    return SsdpServiceInfo(**kwargs)
```

Comment:
Should this be loading the files in the executor?

Reply:
I can easily add... is that something we wish to encourage on all new fixture loads or just ssdp?

Reply:
In general, it should be on all new fixtures since we're loading them in the event loop

Reply:
Isn't this something we should fix more globally? As in, should we make the load async and just do it in the helper function?

I'm also interested how we can make the loading of fixtures more efficient because I'm sure my integrations do IO at every test while we could make that faster by only loading them once, but i never looked into it as i would like to know more on how the correct way is

Reply:
Do we need to distinguish between module constants versus dynamically loaded fixtures? 🤔 

I think it makes sense to load dynamic fixtures via the executor... but I guess module constants would need to be migrated (maybe to pytest fixtures? Or loaded dynamically?)